build.gradle

```
@@ -96,7 +96,7 @@ buildScan {
 
 ext {
     daggerVersion = '2.22.1'
-    fluxCVersion = '1.6.4'
+    fluxCVersion = '1.6.4-beta1'
```

@develric @malinajirka mentioned that the beta tags start with the next version number, so it should be `1.6.5`. Also, there is a `-` between `beta` and the minor number.

I've deleted `1.6.4-beta1` tag and created `1.6.5-beta-1`, so you can just accept the following change and it should work.

```suggestion
    fluxCVersion = '1.6.5-beta-1'
```